types.go

```diff
@@ -161,6 +161,18 @@ type TasksQuery struct {
 	AfterFinishedAt  time.Time
 }
 
+// CancelTasksQuery is a list of filter available to send as query parameters
+type CancelTasksQuery struct {
+	UIDS             []int64
+	IndexUIDS        []string
+	Statuses         []string
+	Types            []string
+	BeforeEnqueuedAt time.Time
+	AfterEnqueuedAt  time.Time
+	BeforeStartedAt  time.Time
+	AfterStartedAt   time.Time
+}
```

Comment:
```suggestion
type CancelTasksQuery struct {
	UIDS             []int64
	IndexUIDS        []string
	Statuses         []string
	Types            []string
	BeforeEnqueuedAt time.Time
	AfterEnqueuedAt  time.Time
	BeforeStartedAt  time.Time
	AfterStartedAt   time.Time
	BeforeFinishedAt time.Time
  AfterFinishedAt time.Time
}
```

Reply:
I didn't add those two filters because they didn't make sense, and they are not in the parameters available in the documentation. Are you sure it's a good think to add it?